manifest.konnector

```diff
@@ -54,8 +54,8 @@
   "langs": ["fr", "en"],
   "locales": {
     "fr": {
-      "short_description": "Récupère vos opérations bancaires",
-      "long_description": "Récupère vos opérations bancaires",
+      "short_description": "Récupère vos opérations bancaires Nalo",
+      "long_description": "Nalo est une société française de gestion de patrimoine accessible sur [https://nalo.fr/](nalo.fr)\n\n**Connectez votre compte Nalo à votre Cozy pour récupérer**\n\n- toutes vos opérations bancaires\n\n**Sauvegardez vos identifiants avec Pass**\n\nEn installant le gestionnaire de mots de passe, vos identifiants sont sauvegardés une bonne fois\npour toutes pour vous faciliter la connexion au site de Nalo. Adieu les mots de passe oubliés !\n\n**Configuration**\n\nPour connecter votre compte, il vous suffit de renseigner dans les champs :\n\n- votre login\n- votre mot de passe\n\nUne fois connecté, votre Cozy se connectera régulièrement à votre compte Nalo pour récupérer les\nnouveaux éléments et les historiser dans votre application Banks\n\nSi vous déconnectez votre compte Nalo, les informations déjà sauvegardées dans votre Cozy restent disponibles. Vous pouvez les supprimer manuellement par ailleurs.",
```

Comment:
@doubleface (nit) Technically as it is a life insurance, we don't retrieve any bank operation, only the account balance.

Reply:
Ok, is it better now ?